nistats/tests/test_first_level_model.py

```diff
@@ -7,20 +7,32 @@
 
 import os
 import shutil
+
```

Comment:
all those changes in the imports look unrelated ? Or is it just because you want to import `with_temp_dir` ?